emmett/tools/auth/apis.py

```diff
@@ -27,10 +27,10 @@
 
 
 class Auth:
-    def __init__(self, app, db, user_model=None):
+    def __init__(self, app, db, user_model=None, membership_model=None, group_model=None, permission_model=None):
```

Comment:
I would change the order in group, membership, permission

Reply:
sure i agree and changed it